README.md

```diff
@@ -23,8 +23,10 @@ Please visit the [documentation].
 | **C**                   | C90, C99, C11       | GCC 6+, clang 3.8+
 | **C++** _(helpers)_[^1] | C++11, C++14, C++17 | GCC 6+, clang 3.8+
 | **Go** _(bindings)_     | 1.9 - 1.12          |
+| **Rust** _(bindings)_[^2] | 1.31.0 and newer  |
```

Comment:
It is 1.31 because we use `edition = 2018` and that's the release introducing it.